src/pybind/mgr/dashboard/ci/cephadm/start-cluster.sh

```diff
@@ -46,6 +46,7 @@ for arg in "$@"; do
   case "$arg" in
     "--dev-mode") DEV_MODE='true'; EXTRA_PARAMS+=" -P dev_mode=${DEV_MODE}" ;;
     "--expanded") EXTRA_PARAMS+=" -P expanded_cluster=true" ;;
+    "NFS=1") EXTRA_PARAMS+=" -P nfs=true" ;;
```

Comment:
@avanthakkar provided we agree that for development purposes and expanded cluster can have several services deployed by default, then this option is redundant as we can use `--expanded`.
In any case, for consistency this should be `--with-nfs` or similar. IMHO `NFS=1` can be misleading.
@epuertat Thoughts?